### DISCUSSION TITLE
TypeError: 'float' object cannot be interpreted as an integer

### DISCUSSION THREAD

**Discussion by Pedrodilamuerte** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#188] [Q&A]
Hello, I'm trying to use the extension but I get the "TypeError: 'float' object cannot be interpreted as an integer" error. It doesn't appear when I use "keep image input size" though. Couldn't find any advice on that. Best regards !

![image](https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111/assets/77626729/2489e37c-fb9f-4386-9beb-ed13da402eba)

**Comment by Kahsolt** [comment]
This is a known bug, but do not know why.
When `upscale ratio` is NOT an integer, the ref image is preprocessed by the webui framework, before passing it to our script.
So we cannot catch the error and do anything. 🤔
But what causes this conflict is still a mystery.

**Reply by Pedrodilamuerte** [reply]
So I should put my upscale ratio either to 1 or 2 ? 

Thanks for your reply
